CRM/Invoicing/Utils.php

```diff
@@ -32,7 +34,7 @@ public static function onToggle($oldValue, $newValue, $metadata) {
     $existingUserViewOptions = civicrm_api3('Setting', 'get', ['return' => 'user_dashboard_options'])['values'][CRM_Core_Config::domainID()]['user_dashboard_options'];
     $optionValues = civicrm_api3('Setting', 'getoptions', ['field' => 'user_dashboard_options'])['values'];
     $invoiceKey = array_search('Invoices / Credit Notes', $optionValues);
-    $existingIndex = in_array($invoiceKey, $existingUserViewOptions);
+    $existingIndex = array_search($invoiceKey, $existingUserViewOptions);
```

Comment:
This was removing key 1 not key 10 - see below we do a strict comparison on FALSE so a return of 0 is fine.

This was the cause of some weird test errors we occasionally saw